astropy/units/__init__.py

```diff
@@ -16,28 +16,31 @@
 from .core import *
 from .quantity import *
 
-from . import si
-from . import cgs
-from . import astrophys
-from . import photometric
-from . import misc
+# isort: split
```

Comment:
These seem to be a bit too many `isort` comments in this file, to the level that it distracts from what is going on. I'd rather have a single `isort off` plus a comment along the lines of "import order matters here -- circular dependencies abound!" (keeping the changes otherwise)

Reply:
✅